spec/acceptance/config_get_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'spec_helper_acceptance'
+require 'fileutils'
+
+describe 'pdk config get' do
+  include_context 'with a fake TTY'
+
+  context 'when run outside of a module' do
+    describe command('pdk config get') do
+      its(:exit_status) { is_expected.to eq 0 }
+      # This setting should appear in all pdk versions
+      its(:stdout) { is_expected.to match(%r{user\.analytics\.user-id=}) }
+      its(:stderr) { is_expected.to be_empty }
```

Comment:
I generally prefer to use `match(%r{\A\Z})` rather than `be_empty` as in the event of a test failure you get a diff showing the contents of the expected empty value rather than `expected empty? to be true, was false`. To that end, I have a custom `have_no_output` matcher in the acceptance suite to wrap this up in an nice DSL (`its(:stderr) { is_expected.to have_no_output }`)

Reply:
Fixed all instances of `empty?`